authentication/views.py

```diff
@@ -1,13 +1,57 @@
 from django.shortcuts import render
 from django.contrib.auth import login,logout,authenticate
-# Create your views here.
+from django.shortcuts import redirect
+from django.contrib.auth.models import User
 
 
 def loginView(request):
-    pass
+    if request.user.is_authenticated :
+        return redirect('index')
+    if request.method == "POST":
+        username = request.POST['username']
+        password = request.POST['password']
```

Comment:
You directly access POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).